index.html

```diff
@@ -289,21 +289,81 @@ <h3><a class="about-connect" href="#">LET'S CONNECT</a></h3>
     </section>
 
     <!-- Contact-Form -->
-    <section class="contact-form" id="Contact">
+    <section class="contact-form" id="contact">
       <h2 class="contact-title" >Contact me</h2>
       <p class="contact-info" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
       
       <fieldset class="fieldset">
-        <form action="https://formspree.io/f/xaykrzrn" method="post">
+        <form action="https://formspree.io/f/xaykrzrn" method="post" id="form">
         <ul>
           <li><input type="text" class="form-input" name="name" placeholder="Your Name" maxlength="30" required></li>
-          <li><input type="email" class="form-input" name="email" placeholder="Your Email" required></li>
+          <li><input type="email" class="form-input" id="email" name="email" placeholder="Your Email" required></li>
```

Comment:
- [ ] I see you have added validation for your email @mertkantaroglu, But still I can submit a form containing a capital letter. Please add the validation for preventing users from using an email with an uppercase letter. 
 - [ ] The error message should show up close to the `submit` button or the `input fields`. (i.e error message can show up around the input fields or just around the submit button)